installing/installing_aws/manually-creating-iam.adoc

```diff
@@ -27,8 +27,8 @@ include::modules/manually-create-identity-access-management.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
-* xref:../../updating/updating-cluster-within-minor.adoc#manually-maintained-credentials-upgrade_updating-cluster-within-minor[Updating a cluster using the web console]
-* xref:../../updating/updating-cluster-cli.adoc#manually-maintained-credentials-upgrade_updating-cluster-cli[Updating a cluster using the CLI]
+* xref:../../updating/updating_a_cluster/updating-cluster-web-console.adoc#manually-maintained-credentials-upgrade_updating-cluster-within-minor[Updating a cluster using the web console]
+* xref:../../updating/updating_a_cluster/updating-cluster-cli.adoc#manually-maintained-credentials-upgrade_updating-cluster-cli[Updating a cluster using the CLI]
```

Comment:
Maybe these should be  replaced by a single link to *Preparing to update a cluster with manually maintained credentials*?  That gets folks into the section of the update docs specific to manual-mode IAM, and it avoids having to pick a favorite two of the several how-to-actually-trigger-updates sections.